### PR TITLE
AP-48 adds new junction object & fields, updates JobApplicationTracker permission set to same

### PR DESCRIPTION
**Pull request by off-nadir-insight**
[Jira issue](https://fritz-b.atlassian.net/browse/AP-48?atlOrigin=eyJpIjoiMjQ1ODE1ZTEwZTA1NDU0MThkMGJhZGY0NTk1MmY5NjQiLCJwIjoiaiJ9)

### CODE REVIEW

File: force-app/main/default/objects/Contact_Association__c/Contact_Association__c.object-meta.xml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    <actionOverrides>
+        <actionName>View</actionName>
+        <formFactor>Small</formFactor>
+        <type>Default</type>
+    </actionOverrides>
+    <allowInChatterGroups>false</allowInChatterGroups>
+    <compactLayoutAssignment>SYSTEM</compactLayoutAssignment>
+    <deploymentStatus>Deployed</deploymentStatus>
+    <description>Relationship between contacts and job applications</description>
+    <enableActivities>false</enableActivities>
```
Comment (fritzb100) [inline force-app/main/default/objects/Contact_Association__c/Contact_Association__c.object-meta.xml]: Should we track activities?? Not sure just asking.

File: force-app/main/default/objects/Contact_Association__c/fields/Contact__c.field-meta.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <fullName>Contact__c</fullName>
+    <description>Contact related to the Job Application</description>
+    <externalId>false</externalId>
+    <label>Contact</label>
+    <referenceTo>Contact</referenceTo>
+    <relationshipLabel>ContactAssociations</relationshipLabel>
+    <relationshipName>ContactAssociations</relationshipName>
+    <relationshipOrder>0</relationshipOrder>
+    <reparentableMasterDetail>false</reparentableMasterDetail>
+    <trackTrending>false</trackTrending>
```
Comment (fritzb100) [inline force-app/main/default/objects/Contact_Association__c/fields/Contact__c.field-meta.xml]: If we do not want to delete historical data, we need to allow for re-parenting

**Review by fritzb100 — COMMENTED** [review]
Is there a tab for this object?
Please add flexipage
Add field Role__c to flexipage

File: force-app/main/default/objects/Contact_Association__c/Contact_Association__c.object-meta.xml (hunk 1 above)
Comment (off-nadir-insight) [reply force-app/main/default/objects/Contact_Association__c/Contact_Association__c.object-meta.xml]: I think we'd want to keep the emails/meetings based on the application? @Oizon thoughts?

**Comment by off-nadir-insight** [comment]
> Is there a tab for this object?

@fritzb100 
Instead of a separate tab, I was thinking of putting the information in a dynamic related list on the job application tab. I'll provide a [screenshot in slack](https://cloudcodeacademy.slack.com/archives/C06BVLK5H0X/p1709776943574689?thread_ts=1709767203.308299&cid=C06BVLK5H0X) for reference

File: force-app/main/default/objects/Contact_Association__c/fields/Contact__c.field-meta.xml (hunk 2 above)
Comment (off-nadir-insight) [reply force-app/main/default/objects/Contact_Association__c/fields/Contact__c.field-meta.xml]: updated!

**Comment by fritzb100** [comment]
> > Is there a tab for this object?
> 
> @fritzb100 Instead of a separate tab, I was thinking of putting the information in a dynamic related list on the job application tab. I'll provide a [screenshot in slack](https://cloudcodeacademy.slack.com/archives/C06BVLK5H0X/p1709776943574689?thread_ts=1709767203.308299&cid=C06BVLK5H0X) for reference

@off-nadir-insight Cool sounds good. Thanks

**Review by fritzb100 — APPROVED** [review]

**Pull request merged**